Formula/c/cyme.rb

```diff
@@ -0,0 +1,27 @@
+class Cyme < Formula
+  desc "List system USB buses and devices"
+  homepage "https://github.com/tuna-f1sh/cyme"
+  url "https://github.com/tuna-f1sh/cyme/archive/refs/tags/v1.7.0.tar.gz"
+  sha256 "63ca7c5f473cbefb1fd07ba9b4f9693268d80560a778a001d82fab68bbd1552a"
+  license "GPL-3.0-or-later"
+  head "https://github.com/tuna-f1sh/cyme.git", branch: "main"
+
+  depends_on "rust" => :build
+  depends_on "libusb"
```

Comment:
No linkage on Linux. I wonder if it's using a vendored version instead.

Reply:
[rusb](https://github.com/a1ien/rusb) usually statically links libusb but uses dynamic linkage to work around https://github.com/rust-lang/rust/issues/96943

https://github.com/a1ien/rusb/blob/f4fc94e/libusb1-sys/build.rs#L206-L217

```rs
let statik = {
    if cfg!(target_os = "macos") {
        match std::env::var("LIBUSB_STATIC").unwrap_or_default().as_ref() {
            "" | "0" => false,
            _ => true,
        }
    } else {
        std::env::var("CARGO_CFG_TARGET_FEATURE")
            .map(|s| s.contains("crt-static"))
            .unwrap_or_default()
    }
};
```

https://doc.rust-lang.org/cargo/reference/environment-variables.html

> `CARGO_CFG_TARGET_FEATURE=mmx,sse` — List of CPU [target features](https://doc.rust-lang.org/reference/conditional-compilation.html#target_feature) enabled.

https://doc.rust-lang.org/reference/conditional-compilation.html#target_feature

> An additional feature of `crt-static` is available to the `target_feature` option to indicate that a [static C runtime](https://doc.rust-lang.org/reference/linkage.html#static-and-dynamic-c-runtimes) is available.